setup.sh

```diff
@@ -52,5 +52,8 @@ fi
 echo "[IAM ZERO]"
 echo "[IAM ZERO] All done! To run the example, you can run the following commands:"
 echo "[IAM ZERO]"
+echo "[IAM ZERO] Ensure that we use the iamzero-test user by setting our environment variable as follows"
+echo "[IAM ZERO] export export AWS_PROFILE=iamzero-test"
+echo "[IAM ZERO]"
```

Comment:
Some users might have a different AWS profile configured (maybe they are testing IAM Zero with a personal or work AWS account) so this could be a bit more general:

```suggestion
echo "[IAM ZERO] Make sure that you're using an active AWS profile"
echo "[IAM ZERO] export AWS_PROFILE=iamzero-test"
echo "[IAM ZERO]"
echo "[IAM ZERO] see https://iamzero.dev/docs/getting-started#creating-an-aws-user for more information"
```